Scripts/Services/Dungeons/TheExodusEncounter/Mobiles/ClockworkExodus.cs

```diff
@@ -92,20 +92,24 @@ public static void DistributeRandomArtifact(BaseCreature bc, Type[] typelist)
         {
             int random = Utility.Random(typelist.Length);
             Item item = Loot.Construct(typelist[random]);
-            DistributeArtifact(DemonKnight.FindRandomPlayer(bc), item);
+            DistributeArtifact(bc.RandomPlayerWithLootingRights(), item);
         }
 
         public static void DistributeArtifact(Mobile to, Item artifact)
         {
-            if (to == null || artifact == null)
-                return;
-
-            Container pack = to.Backpack;
+            if (to != null)
+            {
+                Container pack = to.Backpack;
 
-            if (pack == null || !pack.TryDropItem(to, artifact, false))
-                to.BankBox.DropItem(artifact);
+                if (pack == null || !pack.TryDropItem(to, artifact, false))
+                    to.BankBox.DropItem(artifact);
 
-            to.SendLocalizedMessage(502088); // A special gift has been placed in your backpack.
+                to.SendLocalizedMessage(502088); // A special gift has been placed in your backpack.
```

Comment:
Is there not a separate message for dropping in bankbox (when pack is full) ?

Reply:
There is no cliloc that supports that.

Reply:
What about
1111852 -- You have received a special gift!
1079730 -- The item has been placed into your bank box.
OR
1156322 -- A reward of ~1_ITEM~ has been placed in your bank.

Reply:
I have no idea how EA handles it. There are a ton of special/artifact drops with different messages, most of which go to our bank if your pack is full. I know doom artifact drops, there is no special message when this happens.

Reply:
Hard to test but this makes sense - 1156322

Reply:
Lots of dumb people out there, we have to help them understand what happened to their item :)

Reply:
I like the combination of the first two, personally.